Tooltip: update demo description (#1966)
DevExpress/devextreme-demos: JSDemos/Demos/Tooltip/Overview/jQuery/description.md

> The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 

Author: ```suggestion
The Tooltip component can display a tooltip for an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

> The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 

Reviewer: To bind the Tooltip to **any** element.

```suggestion
The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

## Show and Hide the Tooltip

> To show and hide the Tooltip when certain events are triggered, set the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept several events at once as well as an object.

Reviewer: "Several" is a colloquialism. Use "multiple" instead. These words have identical meaning, however, "multiple" is more suitable to documentation.

```suggestion
To show and hide the Tooltip in response to certain events, specify the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept multiple events at once as well as an object.
```

> To hide the Tooltip when a user clicks outside it, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.

Reviewer: ```suggestion
To hide the Tooltip when a user clicks outside its borders, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
```

## Customize and Animate the Tooltip

> Assign Tooltip's content in the HTML markup. You can also use [contentTemplate](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.

Reviewer: Same as above.

```suggestion
Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```

> If you need to position the Tooltip against a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.

Author: ```suggestion
If you need to position the Tooltip at a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
```

> To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.

Reviewer: Articles.

```suggestion
To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify the [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.
```
